ale/drivers/mro_drivers.py

```diff
@@ -5,9 +5,195 @@
+            if not self.flipped_framelets:
+                time += (adjusted_framelet - 1) * self._framelet_rate
+            else:
+                time += (self._num_framelets - adjusted_framelet) * self._framelet_rate
+            times.append(time)
+        return times
+
+    @property
+    def start_time(self):
+        if not hasattr(self, "_start_time"):
+            start_time = super().ephemeris_start_time
+            start_time -= ((self.exposure_duration / 1000.0) / 2.0)
+            self._start_time = start_time
+        return self._start_time
+
+    @property
+    def ephemeris_start_time(self):
+        if not hasattr(self, "_ephemeris_start_time"):
+            if not self.flipped_framelets:
+                line = 0.5
+            else:
+                line = self.label["IsisCube"]["Core"]["Dimensions"]["Lines"] + 0.5
+            self._ephemeris_start_time = min(self.compute_marci_time(line))
+        return self._ephemeris_start_time
+
+    @property
+    def ephemeris_stop_time(self):
+        if not hasattr(self, "_ephemeris_stop_time"):
+            if not self.flipped_framelets:
+                line = self.label["IsisCube"]["Core"]["Dimensions"]["Lines"] + 0.5
+            else:
+                line = 0.5
+            self._ephemeris_stop_time = max(self.compute_marci_time(line))
+        return self._ephemeris_stop_time
+
+    @property
+    def detector_center_line(self):
+        return 0
+
+    @property
+    def detector_center_sample(self):
+        return 0
+
+    @property
+    def focal2pixel_samples(self):
+        """
+        Expects ikid to be defined. This must be the integer Naif id code of the instrument
+
+        Returns
+        -------
+        : list<double>
+          focal plane to detector samples
+        """
+        return list(spice.gdpool('INS{}_ITRANSS'.format(self.base_ikid), 0, 3))
+
+    @property
+    def focal2pixel_lines(self):
+        """
+        Expects ikid to be defined. This must be the integer Naif id code of the instrument
+
+        Returns
+        -------
+        : list<double>
+          focal plane to detector lines
+        """
+        return list(spice.gdpool('INS{}_ITRANSL'.format(self.base_ikid), 0, 3))
+
+    @property
+    def naif_keywords(self):
+        """
+        Adds the focal length cover keyword to the already populated naif keywords
+
+        Returns
+        -------
+        : dict
+          Dictionary of keywords and values that ISIS creates and attaches to the label
+        """
+        return {**super().naif_keywords, **util.query_kernel_pool(f"*{self.base_ikid}*")}
+
+    @property
+    def sensor_name(self):
+        """
+        ISIS doesn't propergate this to the ingested cube label, so hard-code it.
+        """
+        return "COLOR IMAGER CAMERA"
+
+    @property
+    def sensor_model_version(self):
+        """
+        Returns
+        -------
+        : int
+          ISIS sensor model version
+        """
+        return 1
```

Comment:
Probably be useful to have some boilerplate for docs here, but adding a boilerplate doc string is probably better than nothing?